app/controllers/quotes_controller.rb

```diff
@@ -47,4 +51,12 @@ def destroy
   def quote_params
     params.require(:quote).permit(:body, :quoted_person, :location)
   end
+
+  def vote_types
```

Comment:
This seems weird to me here. It would make more sense inthe actually vote model. And if this is not likely to change dymanically it could be made into a constant and frozen